### ISSUE TITLE
Sass error with Gonzales and indentation rule

### ISSUE THREAD

**Issue by DanPurdy** [sasstools/sass-lint#271]
So when working on the `force-pseudo-nesting` rule I've noticed an issue with the indentation rule which is actually mainly a `gonzales-pe` issue that we should be aware of. 

Would be worth running this through the latest dev build to see if it's still an issue before I report it to the `gonzales-pe` repo, can do this tomorrow.

Anyway:

```scss
.parent
  .child
    p::first-line
      color: #ff0000
```

reports as the following in gonzales

```
{ type: 'class',
  content:
   [ { type: 'ident',
       content: 'parent',
       syntax: 'sass',
       start: { line: 26, column: 2 },
       end: { line: 26, column: 7 },
       indexHasChanged: [ 0 ] } ],
  syntax: 'sass',
  start: { line: 26, column: 1 },
  end: { line: 26, column: 7 },
  indexHasChanged: [ 0 ] }
{ type: 'class',
  content:
   [ { type: 'ident',
       content: 'child',
       syntax: 'sass',
       start: { line: 27, column: 4 },
       end: { line: 27, column: 8 },
       indexHasChanged: [ 0 ] } ],
  syntax: 'sass',
  start: { line: 27, column: 3 },
  end: { line: 27, column: 8 },
  indexHasChanged: [ 0 ] }
```

notice that it misses everything after the child selector.

This results in the indentation rule failing too

```
TypeError: Cannot read property 'is' of undefined
  at /Users/dan/.atom/packages/linter-sass-lint/node_modules/sass-lint/lib/rules/space-before-colon.js:17:21
  at RootNode.traverseByType (/Users/dan/.atom/packages/linter-sass-lint/node_modules/sass-lint/node_modules/gonzales-pe/lib/node/root-node.js:1:1739)
  at RootNode.traverseByTypes (/Users/dan/.atom/packages/linter-sass-lint/node_modules/sass-lint/node_modules/gonzales-pe/lib/node/root-node.js:1:1937)
  at Object.module.exports.detect (/Users/dan/.atom/packages/linter-sass-lint/node_modules/sass-lint/lib/rules/space-before-colon.js:13:9)
  at /Users/dan/.atom/packages/linter-sass-lint/node_modules/sass-lint/index.js:44:27
```

parses fine with the `scss` format

Sass-lint: dev@latest

**Comment by DanPurdy** [comment]
This confirmed to be working gonzales-pe 3.2.1

reporting the following

```
Node {
  type: 'stylesheet',
  content:
   [ Node {
       type: 'ruleset',
       content:
        [ Node {
            type: 'selector',
            content:
             [ Node {
                 type: 'class',
                 content:
                  [ Node {
                      type: 'ident',
                      content: 'parent',
                      syntax: 'sass',
                      start: { line: 1, column: 2 },
                      end: { line: 1, column: 7 } } ],
                 syntax: 'sass',
                 start: { line: 1, column: 1 },
                 end: { line: 1, column: 7 } } ],
            syntax: 'sass',
            start: { line: 1, column: 1 },
            end: { line: 1, column: 7 } },
          Node {
            type: 'space',
            content: '\n',
            syntax: 'sass',
            start: { line: 1, column: 8 },
            end: { line: 1, column: 8 } },
          Node {
            type: 'block',
            content:
             [ Node {
                 type: 'space',
                 content: '  ',
                 syntax: 'sass',
                 start: { line: 2, column: 1 },
                 end: { line: 2, column: 2 } },
               Node {
                 type: 'ruleset',
                 content:
                  [ Node {
                      type: 'selector',
                      content:
                       [ Node {
                           type: 'class',
                           content:
                            [ Node {
                                type: 'ident',
                                content: 'child',
                                syntax: 'sass',
                                start: { line: 2, column: 4 },
                                end: { line: 2, column: 8 } } ],
                           syntax: 'sass',
                           start: { line: 2, column: 3 },
                           end: { line: 2, column: 8 } } ],
                      syntax: 'sass',
                      start: { line: 2, column: 3 },
                      end: { line: 2, column: 8 } },
                    Node {
                      type: 'space',
                      content: '\n',
                      syntax: 'sass',
                      start: { line: 2, column: 9 },
                      end: { line: 2, column: 9 } },
                    Node {
                      type: 'block',
                      content:
                       [ Node {
                           type: 'space',
                           content: '    ',
                           syntax: 'sass',
                           start: { line: 3, column: 1 },
                           end: { line: 3, column: 4 } },
                         Node {
                           type: 'ruleset',
                           content:
                            [ Node {
                                type: 'selector',
                                content:
                                 [ Node {
                                     type: 'typeSelector',
                                     content:
                                      [ Node {
                                          type: 'ident',
                                          content: 'p',
                                          syntax: 'sass',
                                          start: { line: 3, column: 5 },
                                          end: { line: 3, column: 5 } } ],
                                     syntax: 'sass',
                                     start: { line: 3, column: 5 },
                                     end: { line: 3, column: 5 } },
                                   Node {
                                     type: 'pseudoElement',
                                     content:
                                      [ Node {
                                          type: 'ident',
                                          content: 'first-line',
                                          syntax: 'sass',
                                          start: { line: 3, column: 8 },
                                          end: { line: 3, column: 17 } } ],
                                     syntax: 'sass',
                                     start: { line: 3, column: 6 },
                                     end: { line: 3, column: 17 } } ],
                                syntax: 'sass',
                                start: { line: 3, column: 5 },
                                end: { line: 3, column: 17 } },
                              Node {
                                type: 'space',
                                content: '\n',
                                syntax: 'sass',
                                start: { line: 3, column: 18 },
                                end: { line: 3, column: 18 } },
                              Node {
                                type: 'block',
                                content:
                                 [ Node {
                                     type: 'space',
                                     content: '      ',
                                     syntax: 'sass',
                                     start: { line: 4, column: 1 },
                                     end: { line: 4, column: 6 } },
                                   Node {
                                     type: 'declaration',
                                     content:
                                      [ Node {
                                          type: 'property',
                                          content:
                                           [ Node {
                                               type: 'ident',
                                               content: 'color',
                                               syntax: 'sass',
                                               start: { line: 4, column: 7 },
                                               end: { line: 4, column: 11 } } ],
                                          syntax: 'sass',
                                          start: { line: 4, column: 7 },
                                          end: { line: 4, column: 11 } },
                                        Node {
                                          type: 'propertyDelimiter',
                                          content: ':',
                                          syntax: 'sass',
                                          start: { line: 4, column: 12 },
                                          end: { line: 4, column: 12 } },
                                        Node {
                                          type: 'space',
                                          content: ' ',
                                          syntax: 'sass',
                                          start: { line: 4, column: 13 },
                                          end: { line: 4, column: 13 } },
                                        Node {
                                          type: 'value',
                                          content:
                                           [ Node {
                                               type: 'color',
                                               content: 'ff0000',
                                               syntax: 'sass',
                                               start: { line: 4, column: 14 },
                                               end: { line: 4, column: 20 } } ],
                                          syntax: 'sass',
                                          start: { line: 4, column: 14 },
                                          end: { line: 4, column: 20 } } ],
                                     syntax: 'sass',
                                     start: { line: 4, column: 7 },
                                     end: { line: 4, column: 20 } },
                                   Node {
                                     type: 'declarationDelimiter',
                                     content: '\n',
                                     syntax: 'sass',
                                     start: { line: 4, column: 21 },
                                     end: { line: 4, column: 21 } } ],
                                syntax: 'sass',
                                start: { line: 4, column: 1 },
                                end: { line: 4, column: 21 } } ],
                           syntax: 'sass',
                           start: { line: 3, column: 5 },
                           end: { line: 4, column: 21 } } ],
                      syntax: 'sass',
                      start: { line: 3, column: 1 },
                      end: { line: 4, column: 21 } } ],
                 syntax: 'sass',
                 start: { line: 2, column: 3 },
                 end: { line: 4, column: 21 } } ],
            syntax: 'sass',
            start: { line: 2, column: 1 },
            end: { line: 4, column: 21 } } ],
       syntax: 'sass',
       start: { line: 1, column: 1 },
       end: { line: 4, column: 21 } } ],
  syntax: 'sass',
  start: { line: 1, column: 1 },
  end: { line: 4, column: 21 } }
```

**Comment by DanPurdy** [comment]
This is resolved in sass-lint 1.6.0 and gonzales-pe 3.2.x
